cvrf/suse/suse.go

```diff
@@ -79,9 +80,16 @@ func (c Config) update(os string, urls []string) error {
 	for _, cvrfXml := range cvrfXmls {
 		var cv Cvrf
 		if len(cvrfXml) == 0 {
-			log.Printf("empty CVRF xml: %s", cv.Tracking.ID)
+			log.Println("empty CVRF xml")
 			continue
 		}
+
+		if !utf8.Valid(cvrfXml) {
+			log.Println("invalid UTF-8")
+			s := strings.ToValidUTF8(string(cvrfXml), "")
+			cvrfXml = []byte(s)
```

Comment:
inline
```suggestion
			cvrfXml = []byte(strings.ToValidUTF8(string(cvrfXml), ""))
```

Reply:
Just my rule: No more than two processes in one line for readability
1. string(cvrfXml)
1. strings.ToValidUTF8(b)
1. []byte(s)

But it is not so strong rule, so I follow your suggestion here.